Dockerfile

```diff
@@ -1,6 +1,10 @@
 FROM ubuntu:14.04
 MAINTAINER PhenoMeNal-H2020 Project <phenomenal-h2020-users@googlegroups.com>
-LABEL Description="Galaxy test for running inside Kubernetes."
+
+LABEL Description="Galaxy 16.07-phenomenal for running inside Kubernetes."
+LABEL software="Galaxy"
+LABEL software.version="16.07-pheno"
+LABEL version="0.1"
```

Comment:
Sorry, I was ambiguous :-)  I meant using a single LABEL command, not a single label.  From what I understand this should generate a single layer and set the same labels:
```
LABEL Description="Galaxy 16.07-phenomenal for running inside Kubernetes."  \
      software="Galaxy" \
      software.version="16.07-pheno" \
      version="0.1"
```